manifest_staging/charts/gatekeeper/templates/gatekeeper-validating-webhook-configuration-validatingwebhookconfiguration.yaml

```diff
@@ -48,6 +48,24 @@ webhooks:
     {{- end }}
     resources:
     - '*'
+    # Explicitly list all known subresources except "status" (to avoid destabilizing the cluster and increasing load on gatekeeper).
+    # You can find a rough list of subresources by doing a case-sensitive search in the Kubernetes codebase for 'Subresource("'
+    - 'pods/ephemeralcontainers'
```

Comment:
Thank you for adding this @mac-chaffee 🎉 

Do we want to create a parameter for this so people can opt-in or opt-out? Should this be turned on by default?